### PR TITLE
8299296: Write a test to verify the components selection sends ItemEvent

### PR DESCRIPTION
**Pull request by ravigupta30**
This testcase verifies the following assertions

a. Selecting an item in the Choice using mouse generates ItemEvent.
b. Single-click on an Checkbox generates ItemEvent.

Testing:
Tested using Mach5(20 times per platform) in macos, linux and windows and got all pass.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8299296](https://bugs.openjdk.org/browse/JDK-8299296): Write a test to verify the components selection sends ItemEvent

### Reviewers
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11786/head:pull/11786` \
`$ git checkout pull/11786`

Update a local copy of the PR: \
`$ git checkout pull/11786` \
`$ git pull https://git.openjdk.org/jdk pull/11786/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11786`

View PR using the GUI difftool: \
`$ git pr show -t 11786`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11786.diff">https://git.openjdk.org/jdk/pull/11786.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/awt/event/ComponentEvent/ComponentItemEventTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.*;
+import java.awt.event.*;
```
Comment (mrserb) [inline test/jdk/java/awt/event/ComponentEvent/ComponentItemEventTest.java:25]: Just a small suggestion: "use single-class imports"

**Review by mrserb — APPROVED** [review]

**Comment by ravigupta30** [comment]
Any volunteers for a sponsor?

**Comment by ravigupta30** [comment]
@mrserb requesting you to please sponsor.

**Pull request closed without merging**